01-main/packages/flameshot

```diff
@@ -1,5 +1,5 @@
 DEFVER=1
-CODENAMES_SUPPORTED="buster bullseye trixie focal jammy mantic noble oracular"
+CODENAMES_SUPPORTED="buster bullseye trixie focal sid jammy mantic noble oracular"
```

Comment:
```suggestion
CODENAMES_SUPPORTED="bookworm jammy noble"
```
They only have [release assets](https://github.com/flameshot-org/flameshot/releases/latest) for Debian 12 and Ubuntu 22.04 and 24.04.